src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        String color = String.valueOf(colorSupplier.getRandomColor());
+        int number = random.nextInt(100);

```

Comment:
All magic numbers in your code should be constants. You should define '100' as a constant with a clear name at the class level to improve code readability and maintainability.